test/functional/page_objects/header_page.js

```diff
@@ -169,6 +169,30 @@ export function HeaderPageProvider({ getService, getPageObjects }) {
         .findByLinkText(quickTime).click();
     }
 
+    async getAutoRefreshState() {
+      return testSubjects.getAttribute('globalTimepickerAutoRefreshButton', 'data-test-subj-state');
+    }
+
+    // check if the auto refresh state is active and to pause it
+    async pauseAutoRefresh() {
+      let result = false;
+      if (await this.getAutoRefreshState() === 'active') {
+        await testSubjects.click('globalTimepickerAutoRefreshButton');
+        result = true;
+      }
+      return Promise.resolve(result);
```

Comment:
This shouldn't need the `Promise` since the method is marked `async`. Same for `resumeAutoRefresh()`